src/foundryvtt-gmScreen.ts

```diff
@@ -89,3 +89,31 @@ Hooks.on('renderJournalDirectory', (app, html, data) => {
     _addGmScreenButton(html);
   }
 });
+
+Hooks.on('renderGmScreenApplication', (app,html,data) => {

```

Comment:
`(app,html,data)` doesn't look like prettier ran correctly, should have spaces after each comma.